src/Goerli-DssSpell.sol

```diff
@@ -70,8 +71,10 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardCollaterals();
+        onboardCollaterals();
+        // updateCollaterals();
         // offboardCollaterals();
+        DssExecLib.setChangelogVersion("1.14.4");
```

Comment:
```suggestion
        DssExecLib.setChangelogVersion("1.14.4");
```

Currently deployed version is 1.14.2 consistent across Goerli and Mainnet. PE-1102 (yet to be cast) does not bump version. Bump to 1.14.2 was on 4th October 2022 spell; should this be 1.14.4 or 1.14.3?